shared/events.js

```diff
@@ -377,9 +377,9 @@ export class DeleteAttribute extends HashableAction {
 
 export class MailboxContract extends HashableContract {
   static vuex = MailboxContract.Vuex({
-    state: { messages: [] },
+    state: { messages: [], nextId: 0 },
     mutations: {
-      PostMessage (state, data) { state.messages.push(data) },
+      PostMessage (state, data) { state.messages.push({...data, id: state.nextId++}) },
       PostInvite (state, data) { state.messages.push(data) },
```

Comment:
PostInvite is outdated an needs to be deleted. I'm not sure about this `nextId`, can you add above it on line 380 `TODO: verify the correctness of 'nextId' in more detail later`?